dracut/module-setup.sh

```diff
@@ -95,6 +95,11 @@ install() {
     inst_simple "${doc}" "/usr/share/docs/${relative}"
   done <<<"$( find "${zfsbootmenu_module_root}/help-files" -type f )"
 
+  compat_dirs=( "/etc/zfs/compatibility.d" "/usr/share/zfs/compatibility.d/" )
+  for compat_dir in "${compat_dirs[@]}"; do
+    # shellcheck disable=2164
+    [ -d "${compat_dir}" ] && tar -cf - "${compat_dir}" | ( cd "${initdir}" ; tar xfp - )
```

Comment:
I  couldn't find a `dracut` function that did this properly. Since we're just copying a few text files, a tar pipeline is sufficient for this.